### ISSUE TITLE
UI Change Request: Remove the word "Download:" from the download page list

### ISSUE THREAD

**Issue by PiotrKrzyzek** [bbilly1/tubearchivist#64]
On the tubearchivist download page (ie mydomain.com/downloads/), the download is shows nicely and has great info though the video title has "Download:" prepended to it.

It seems redundant to have that in the video title as we know we're on the downloads page. Though it also makes it a bit harder to read the titles.

**Comment by bbilly1** [comment]
Yes, I get what you mean with redundant. I added that to separate between "ignore" and "download", so the list switches once you toggle "Show only ignored videos". I didn't want to create a separate page like /ignore/ just for that. But the title changes also from "Download queue" to "Ignored from download" once you flip the toggle...

Do you think that's obvious enough? Or maybe there is a better way to do that without decreasing the readability of the titles. Any ideas?

**Comment by PiotrKrzyzek** [comment]
Visual tags overlayed on the thumbnail might be a way to do it. Or even a 'tags' / data 'row' right below the title div.

If you do a visual overlay 'tag', then using the :before or :after property might work nicely for something simple.

Another option might be yet another column with an icon to let users know what 'state' the video is in on that specific page.

For the same of example, let's use DataTables JS as a base for the following. I'm using that library as an example as we've using on a project right now so it's fresh in my mind:

If we change the downloads page to use a table like format, then we could have a lot more control of what's going on in this page. We could have a Bulk Actions bar (section) up-top, then filters and search below it, and then the sortable / configurable table with all the data right under it. As since DataTables JS can easily run via AJAX and JSON queries it'd be easy to update the data 'live' in there to make it dynamic (both with or without reloading the page). On the left we'd have our thumbnail column, then the middle column would be the same as now, and then THREE right columns for DOWNLOAD, IGNORE, DELETE.

The above is only an idea not a full on suggestion and I'm not sure if a 'table' would be better or not.

I did take the liberty to mock up some ideas in screenshot below. I hope they can be helpful to you. But before that: as the "show ignored" toggle currently does fully toggle a display change between ONLY ignore and ONLY "yet to do things with", maybe the solution is to simply have a small little notice up-top that displays when 'show ignored' is turned on?

Either way, some screenshots:

<img width="1031" alt="Screen Shot 2021-10-21 at 2 51 37 PM" src="https://user-images.githubusercontent.com/602636/138347310-b34007f2-1d37-4541-9f6a-09309ecba59b.png">
<img width="1022" alt="Screen Shot 2021-10-21 at 2 49 05 PM" src="https://user-images.githubusercontent.com/602636/138347313-c6cc3dc1-2361-459a-81be-42b1d6d111f9.png">
<img width="1047" alt="Screen Shot 2021-10-21 at 2 47 19 PM" src="https://user-images.githubusercontent.com/602636/138347314-d11867b4-1507-4684-ab6b-0f8712ff53b1.png">

Don't mind the poor positioning. I just did something quick in-between calls. :)

Though looking more and more of this: with the 'ignore' toggle never showing BOTH, I think simply removing the text "Download" and "Ignore" would be the quickest tweak.

Quick side note: in the screenshots I also increased the title font-size (of those list h3's) to 1.3em as it make it a little bit easier to read on my old man eyes.
